### DISCUSSION TITLE
[BUG] Zustand produce a ts error when using combine with persist

### DISCUSSION THREAD

**Discussion by judicaelandria** [pmndrs/zustand#1264] [Q&A] [answered]
[Example](https://codesandbox.io/s/blissful-sea-p146jz?file=/store.ts)
It's working in v 3.6.8 but not anymore in the latest version which is 4.1.1. and all the types are unknown.

**Comment by dai-shi** [comment, marked as answer]
Please do something like this: https://codesandbox.io/s/magical-wildflower-j9ujzd?file=/src/store.ts

If you want to split store creator, please read the docs very carefully.

**Reply by judicaelandria** [reply]
I did! Ts was happy when the version I showed to you below, that's why I thought it was a bug from zustand, but my bad sorry about that!
